fe/fe-core/src/main/java/com/starrocks/http/WebUtils.java

```diff
@@ -161,4 +171,48 @@ private static char decodeHexNibble(final char c) {
             return Character.MAX_VALUE;
         }
     }
+
+    /**
+     * This method sanitizes the input URI by replacing the values of certain query parameters with "*".
+     * The parameters to be sanitized are listed in `SANITIZED_HTTP_PARAMS`.
+     * If these parameters are not present in the URI, the method returns the original URI.
+     *
+     * @param uri The input URI to be sanitized.
+     * @return The sanitized URI with the values of certain query parameters replaced with "*".
+     * @throws URISyntaxException If the input string can't be parsed as a URI reference.
+     */
+    public static String sanitizeHttpReqUri(String uri) throws URISyntaxException {
```

Comment:
wondering if can leverage QueryStringDecoder to simplify some code.

Reply:
not sure, chatgpt told me to do it this way...